src/main/java/uk/gov/hmcts/reform/et/service/ServiceHearingsService.java

```diff
@@ -40,21 +43,21 @@ public ServiceHearingValues getServiceHearingValues(String authorization, Servic
         CaseData caseData = CaseDataMapping.mapCaseData(caseDetails.getData());
 
         log.info("Mapping hearing values for Case id : {}, generating Service Hearing Values", caseDetails.getId());
-        return mapServiceHearingValues(caseDetails.getCaseTypeId(), caseData);
+        return mapServiceHearingValues(caseDetails, caseData);
     }
 
-    private ServiceHearingValues mapServiceHearingValues(String caseTypeId, CaseData caseData) {
+    private ServiceHearingValues mapServiceHearingValues(CaseDetails caseDetails, CaseData caseData) {
```

Comment:
If we're going to pass in the whole caseDetails object now can we remove the caseData param since caseDetails contains caseData?

Reply:
Good point. Refactored to pass only the data it needs